services/resource-service/server/src/main/java/org/apache/airavata/mft/resource/server/backend/file/FileBasedResourceBackend.java

```diff
@@ -383,4 +385,75 @@ public boolean updateDropboxResource(DropboxResourceUpdateRequest request) throw
     public boolean deleteDropboxResource(DropboxResourceDeleteRequest request) throws Exception {
         throw new UnsupportedOperationException("Operation is not supported in backend");
     }
+
+    @Override
+    public Optional<FTPResource> getFTPResource(FTPResourceGetRequest request) throws Exception {
+        InputStream inputStream = FileBasedResourceBackend.class.getClassLoader().getResourceAsStream("resources.json");
```

Comment:
Minor Suggestion: instead of hard coding the name of resource file use field resourceFile

Reply:
Done I have changed it

Reply:
Since this branch is taken out of feature/ftp-transport. I have made the changes there as it is related to that I am attaching my commit here
https://github.com/apache/airavata-mft/pull/13/commits/68897a10d99e776aaa2dd8bccce13287fb3e7783